packages/serialize-error/lib/index.js

```diff
@@ -61,6 +63,11 @@ function serializeError(error) {
 		errorProperties.relatesToSystems = error.relatesToSystems;
 	}
 
+	// If set, error cause (which in turn is an error instance) is serialized
+	if (error.cause && error.cause instanceof Error) {
+		errorProperties.cause = serializeError(error.cause);
```

Comment:
**thought:** maybe down the line we need to think about whether to limit the recursion here? I don't think in this PR but I guess it's a possible vector for errors to creep in. Though tbh I think you'd have to try quite hard to make this infinitely recurse 🤷 anyway not something to address now